CS361Project/urls.py

```diff
@@ -21,6 +21,7 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('login/', Login.as_view()),
```

Comment:
What is the purpose of changing the url for login? If it is not necessary to change it, I think it should be kept the same so that other parts of the code work as intended.

Reply:
The logout button leads back to the login page and requires the user to login again. This is why I included that path. The path for running the program is also still there so the login page works just fine both ways!

Reply:
I see. I had thought you had changed the original one instead of just adding another path. Thanks.